turbinia/e2e/run-e2e-gke.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+# Turbinia GKE e2e tests
+# The e2e test will test Turbinia googleclouddisk processing
+# Requirements:
+# - have 'kubectl', 'jq' packages installed
+# - have the Turbinia Helm chart deployed and are authenticated to the GKE cluster
+# - have the 'turbinia-client' CLI installed
+
+set -o posix
+set -e
+
+RELEASE="test"
+DISK="disk-1"
+FAILED=0
+REQUEST_ID=$(uuidgen -rt)
+DATE=$(date -I)
+
+if [ $# -ne  2 ]
+then
+  echo "Not enough arguments supplied, please provide GCP project and zone"
+  echo "$0 [PROJECT] [ZONE]"
+  exit 1
+fi
+
+GCP_PROJECT="$1"
+GCP_ZONE="$2"
+
+echo -n "Started at "
+date -Iseconds
+
+# Back up existing Turbinia config else script will attempt to connect to wrong Turbinia instance
+if  [ -f ~/.turbinia_api_config.json ]
+then
+  echo "Backing up existing Turbinia config to ~/.turbinia_api_config.json.$DATE"
+  mv ~/.turbinia_api_config.json ~/.turbinia_api_config.json.$DATE
+fi
+
+# Replace Turbinia config with test config
+echo "Writing turbinia config to $HOME/.turbinia_api_config.json..."
+cat > $HOME/.turbinia_api_config.json <<EOL
+{
+	"default": {
+		"description": "Turbinia client test config",
+		"API_SERVER_ADDRESS": "http://127.0.0.1",
+		"API_SERVER_PORT": 8000,
+		"API_AUTHENTICATION_ENABLED": false,
+		"CLIENT_SECRETS_FILENAME": ".client_secrets.json",
+		"CREDENTIALS_FILENAME": ".credentials_default.json"
+	}
+}
+EOL
+
+# Turbinia e2e test
+echo "Starting e2e test for Turbinia..."
+
+# Forward k8s services
+echo "Forwarding Turbinia API k8s $RELEASE service"
+kubectl --namespace default port-forward service/$RELEASE-turbinia 8000:8000  > /dev/null 2>&1 &
+# Give time before submitting request to service
+sleep 5
+
+# List Turbinia config
+echo "Listing Turbinia config..."
+turbinia-client config list
+
+# Run Turbinia googleclouddisk processing
+echo "Running Turbinia: turbinia-client submit googleclouddisk --project $GCP_PROJECT --zone $GCP_ZONE --disk_name $DISK --request_id $REQUEST_ID"
+turbinia-client submit googleclouddisk --project $GCP_PROJECT --zone $GCP_ZONE --disk_name $DISK --request_id $REQUEST_ID
+# Give time for Tasks to populate
+sleep 5
```

Comment:
I updated the while statement slightly to evaluate on while $req_status = "running".

The other reason I did this is because the time you submit the request has lag time to when its actually reached the server and is stored in redis so I'm sleeping here to wait a little, maybe a better way to do this so open to ideas. I think if we had polling implemented directly then this wouldn't be much of an issue